resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/AsyncResponseConsumer.java

```diff
@@ -288,11 +293,15 @@ public void onSubscribe(Subscription subscription)
       public void subscribe(Object rtn)
       {
          @SuppressWarnings({ "unchecked", "rawtypes" })
-         Publisher<?> stage = ((AsyncStreamProvider)asyncStreamProvider).toAsyncStream(rtn);
-         stage.subscribe(this);
+         Publisher<?> publisher = ((AsyncStreamProvider)asyncStreamProvider).toAsyncStream(rtn);
+         publisher.subscribe(this);
       }
    }
 
+   /*
+    * This class is not currently used, having been replaced by AsyncStreamSseResponseConsumer even for 
+    * non-SSE connections.
+    */
    private static class AsyncStreamingResponseConsumer extends AsyncStreamResponseConsumer 
```

Comment:
Is this still really the case? It looks needed above

Reply:
@asoldano, ah, yes, it's back in use.

Reply:
OK, thanks